src/OGL3X/Shaders_ogl3x.h

```diff
@@ -150,12 +150,14 @@ MAIN_SHADER_VERSION
 
 static const char* fragment_shader_header_common_variables =
 MAIN_SHADER_VERSION
+#ifndef GLESX
```

Comment:
Hello Gonetz, it seems to be working without that defined. Are there any specific tests I can run to verify that mipmap is working correctly? I did test LOD by using the peach painting in Super Mario 64, that went from crashing to working correctly afterwards.